src/files-breadcrumbs.tsx

```diff
@@ -105,12 +105,12 @@ export function FilesBreadcrumbs({ path, showHidden, setShowHidden }: { path: st
                     />}
                 {!editMode && fullPath.map((dir, i) => {
                     return (
-                        <React.Fragment key={dir || "/"}>
+                        <React.Fragment key={`${dir}-${i}`}>
```

Comment:
`i` must not be part of a key. The point of keys are to uniquely identify the content/identity of an entry *independently* from its position (otherwise React wouldn't need keys in the first place), to know which rows have to be re-rendered. I suggest to build up the key from the full path during the iteration, like "usr", "usr/share", "usr/share/doc", etc. An external accumulator variable should work, or a slice of `path`.

It may work fine in this specific case, but it's a very dangerous practice, so please don't learn it :grin:

Reply:
Sure, it's a bit ugly to have a var defined outside but I don't see a nicer way to do it.

Reply:
And reworked to use fullPath.